bokeh/models/tickers.py

```diff
@@ -66,7 +66,7 @@ class AdaptiveTicker(Ticker):
     The smallest allowable interval between two adjacent ticks.
     """)
 
-    max_interval = Float(100.0, help="""
+    max_interval = Float(float('Inf'), help="""
```

Comment:
Sorry. The problem is since we are sending everything, the max max value actually gets sent, and it stops adaptively ticking after 100 (and then ticks and labels pile up on top of one another). The JS default was `Infinity` which is why it worked. Open to other approaches.